### PR TITLE
[0.8.0] Show status of last run in Jobs list

### PR DESCRIPTION
**Pull request by anukul**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Other (please describe):

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] If it's a backend change, tests for the changes have been added and `go test ./...` runs successfully from the root folder.
- [ ] If it's a frontend change, Prettier has been run
- [ ] Docs have been reviewed and added / updated if needed

## What is the current behavior?

User must click on a Job to see the status of its last run

## What is the new behavior?

For each job, the status of its last run will be shown on the Job List page.

image

Styling
- For failed runs: red, bold
- For successful runs: green, bold
- For running runs: same as version text
- For deleting runs: same as version text

### CODE REVIEW

**Comment by anukul** [comment]
All possible status values:

image

File: dashboard/src/main/home/cluster-dashboard/chart/Chart.tsx [hunk 1]
```diff
@@ -123,7 +158,18 @@ const Chart: React.FunctionComponent<Props> = ({
         </TagWrapper>
       </BottomWrapper>
 
-      <Version>v{release?.version || chart.version}</Version>
+      <TopRightContainer>
+        {isJob && jobStatus?.status && (
+          <>
+            <JobStatus status={jobStatus.status}>
+              Last run {jobStatus.status.toUpperCase()} at{" "}
+              {readableDate(jobStatus.start_time)}
```
Comment (anukul) [inline dashboard/src/main/home/cluster-dashboard/chart/Chart.tsx:166]: Completion time is only set for `succeeded` jobs, so we're using the start time here.

Considering that, do we want to reword this sentence to something else?

Comment (abelanger5) [reply dashboard/src/main/home/cluster-dashboard/chart/Chart.tsx:200]: Yes, let's change to "Last run started at..."?

**Review by abelanger5 — APPROVED** [review]
Live updates are working great @anukul -- this is ready for staging. We'll batch styling updates for this release so I'll make sure changing the wording gets added to the styling fixes.

**Pull request merged**